### ISSUE TITLE
Integral animation

### ISSUE THREAD

**Issue by pedroslrey** [3b1b/manim#862]
I would like to replicate the animations from tha video https://youtu.be/rfG8ce4nNh0?t=491 at the moment 8:11. Does someone knows how to make it? (it don't need to be exactly equal)

**Comment by mattdeitke** [comment]
Yes! Use [manim](https://github.com/3b1b/manim) (that is the animation library that Grant built).

Basically, just draw like 20 rectangles with differing heights and then iteratively move them up and down in a sequence.

:)

**Comment by quark67** [comment]
The code for this video is here: https://github.com/3b1b/manim/blob/master/from_3b1b/old/eoc/chapter8.py
